### PR TITLE
Storage clearing - update tests to cover more edge-cases

### PR DESCRIPTION
**Pull request by kdzwinel**
based on the discussion in https://app.asana.com/0/0/1201727304104963/f

### CODE REVIEW

**Comment by kdzwinel** [comment]
Hey @tomasstrba  if you have time PTAL at those updates to data clearing tests. I added couple of edge-cases and improved descriptions of tests.

File: storage-clearing/tests.json [hunk 1]
```diff
@@ -22,30 +22,51 @@
                 "exceptPlatforms": []
             },
             {
-                "name": "cookie is set on a non-fireproofed subdomain - cookie should be cleared",
-                "cookieDomain": "sub.fireproofed.test",
+                "name": "only subdomain is fireproofed, but cookie is using dot notation - it should not be cleared",
```
Comment (tomasstrba) [inline storage-clearing/tests.json:25]: Konrad, cookie with domain `.no-fire.test` is also sent to `no-fire.test` or `anothersub.no-fire.test`. What it should not me cleared?

**Review by tomasstrba — APPROVED** [review]
LGTM! 👍
